airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -490,15 +490,21 @@ def generate_scd_type_2_model(self, from_table: str, column_names: Dict[str, Tup
     ) as _airbyte_end_at,
     lag({{ cursor_field }}) over (
         partition by {{ primary_key }}
-        order by {{ cursor_field }} desc, _airbyte_emitted_at desc
-    ) is null as _airbyte_active_row,
+        order by {{ cursor_field }} desc, _airbyte_emitted_at desc {{ cdc_updated_at_order }}
+    ) is null {{ cdc_active_row }} as _airbyte_active_row,
```

Comment:
```suggestion
    ) is null {{ cdc_active_row }}as _airbyte_active_row,
```

otherwise it's adding an extra space to all scd files

then you should adapt the line below with an extra space as: 
```
cdc_active_row_pattern = "and _ab_cdc_deleted_at is null "
```